library/src/main/java/com/schibsted/spain/barista/util/ViewTreeAnalyzer.java

```diff
@@ -0,0 +1,24 @@
+package com.schibsted.spain.barista.util;
+
+import android.view.View;
+import android.view.ViewGroup;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class ViewTreeAnalyzer {
+  public static List<View> getAllChildren(View parent) {
+    if (!(parent instanceof ViewGroup)) {
+      return Collections.singletonList(parent);
```

Comment:
This returns the view itself. Doesn't that goes against the method name? I think it should return an empty list.